integration/lsp/smoke_spec.ts

```diff
@@ -54,7 +54,11 @@ describe('Angular Language Service', () => {
     });
     server.on('message', (data: Message) => {
       if (isNotificationMessage(data)) {
-        console.log('[server]', data.params.message);
+        if (data.params && data.params.message) {
+          console.log('[server]', data.method, data.params.message);
+        } else {
+          console.log('[server]', data.method);
+        }
```

Comment:
can we conditionally log on the message? e.g.

```typescript
const toLog = ['[server]', data.method];
if (data.params && data.params.message) toLog.push(data.params.message);
console.log(...toLog);
```

or

```typescript
console.log('[server]', data.method);
if (data.params && data.params.message) console.log(data.params.message);
```

Reply:
Changed to 1st suggestion.